diff --git a/node_modules/pkg/prelude/bootstrap.js b/node_modules/pkg/prelude/bootstrap.js
index 216579e..df9cfae 100644
--- a/node_modules/pkg/prelude/bootstrap.js
+++ b/node_modules/pkg/prelude/bootstrap.js
@@ -210,9 +210,7 @@ function projectToFilesystem (f) {
 
 function projectToNearby (f) {
   return require('path').join(
-    require('path').dirname(
-      EXECPATH
-    ),
+    process.cwd(),
     require('path').basename(
       f
     )
@@ -1406,20 +1404,6 @@ function payloadFileSync (pointer) {
     }
   }
 
-  childProcess.spawn = function () {
-    var args = cloneArgs(arguments);
-    setOptsEnv(args);
-    modifyShort(args);
-    return ancestor.spawn.apply(childProcess, args);
-  };
-
-  childProcess.spawnSync = function () {
-    var args = cloneArgs(arguments);
-    setOptsEnv(args);
-    modifyShort(args);
-    return ancestor.spawnSync.apply(childProcess, args);
-  };
-
   childProcess.execFile = function () {
     var args = cloneArgs(arguments);
     setOptsEnv(args);
